CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed use of current region of an instance if no ec2 region is specified.
```

Comment:
Fix in Cloud AWS plugin: Use the region of the instance if no EC2 region is specified in the configuration.